README.md

```diff
@@ -1,2 +1,15 @@
-# HW01
-Edit the readme
+# _**Hi Everyone!**_
+I am happy to see that you can both bold _**and**_ italicize text in these documents
+ 
+## Biography
+* My name is **Patrick Cantwell** and I am an incoming biochemistry and molecular biophysics Ph.D. student.
+
+* This is a picture of me from a few weeks ago with the longest hair I've ever had
+
+<img src="Photo of Me! 2.0.jpg" width="500" height="400" />
+
+* I luckily got it cut a few days ago for the first time in ~6 months, but I don't have a picture of that.
```

Comment:
A haircut is always a nice feeling! Did you venture outside to a hair salon? Hopefully it was a safe trip!